com.unity.shadergraph/Editor/Data/Graphs/AbstractShaderProperty.cs

```diff
@@ -100,7 +101,7 @@ internal virtual void AppendPropertyBlockStrings(ShaderStringBuilder builder)
 
         internal abstract void ForeachHLSLProperty(Action<HLSLProperty> action);
 
-        internal abstract string GetPropertyAsArgumentString();
+        internal abstract string GetPropertyAsArgumentString(string precisionString);
```

Comment:
I moved the precision string logic outside of GetPropertyAsArgumentString(), to keep the per-property implementation of this function simple.